test/models/IllPersonSchema.test.ts

```diff
@@ -0,0 +1,98 @@
+import { IllPerson } from 'app/models/fields/IllPerson';
+import { schema } from 'app/models/fields/IllPerson.schema';
+import { SchemaValidator } from 'app/utils/validation/SchemaValidator';
+import { ValidationError } from 'app/utils/validation/ValidationError';
+
+import { assertValidationErrors } from 'test/models/ValidationAssertions';
+
+const validator = new SchemaValidator(schema);
+const emptySelectionErrorMessage = 'You must select a person';
+const emptyOtherPersonErrorMessage = 'You must tell us more information';
+
+describe('IllPerson schema', () => {
+
+    describe('should reject', () => {
+
+        it('empty object with only one error', () => {
```

Comment:
The name of these tests doesn't make much sense within the context of the code because of the `it() `functon. Id start it these names with `should....`

I would also rename the describe they are all within. Id be happy with just `rejects`

Reply:
If you could fix that through this code that would be great <3

Reply:
I see what you mean. However these changes would lead to the following output when running the tests:
```
IllPerson schema
    rejects
        should reject empty object with only one error
```
for example. Are we prioritising readability of code or test output?

Reply:
Im sure we can do both, I` prefer should reject empty object with only one error`, maybe rename the 'rejects' description to something else?

Reply:
How's this? :) Think "invalid" and "valid" look more like subtitles than bad sentence structure so this works.

![Screenshot 2020-08-04 at 10 46 20](https://user-images.githubusercontent.com/43062389/89279829-be2b1680-d63f-11ea-8988-66a6096bb0fe.png)

Reply:
Im going to hang that up my wall <3